spec/requests/providers/proceeding_merits_task/is_client_biological_parent_controller_spec.rb

```diff
@@ -73,9 +73,9 @@ module ProceedingMeritsTask
             expect(legal_aid_application.legal_framework_merits_task_list).to have_not_started_task(:PB003, :client_relationship_to_proceeding)
           end
 
-          it "redirects to next page" do
+          it "redirects to parental responsibility question page" do
             patch_request
-            expect(response).to have_http_status(:redirect)
+            expect(response).to redirect_to providers_merits_task_list_does_client_have_parental_responsibility_path(proceeding)
```

Comment:
Is the pattern to now expect the `http_status` instead?

Reply:
@naseberry Yes, except that in this case, we're using an explicit `redirect_to` in the controller (line 10), rather than using the flow step, so I thought the test should be explicit